staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy.go

```diff
@@ -53,6 +54,9 @@ func (a statusStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set
 	return fields
 }
 
+func isLocationDiffSyncerAnnotation(annotationKey string) bool {
```

Comment:
move into patch file

Reply:
Done in 5c1727bc7f07a2227c0c180b9c924156a7b72146